presentation/matrix_view.py

```diff
@@ -13,10 +11,16 @@
 GREEN = "rgb(0,255,0)"
 
 
-def make_mesh_test_matrix_layout(mesh: MeshResults, config: Config) -> dcc.Graph:
+def make_mesh_test_matrix_layout(mesh: MeshResults, metric: str, config: Config) -> Dict:
     matrix = Matrix(mesh)
-    data = make_data(mesh, matrix, config.latency)
-    annotations = make_annotations(mesh, matrix)
+    if metric == "latency":
```

Comment:
Please extract `get_thresholds(metric) thresholds` function and use it in make_data() call

Reply:
Done